### PR TITLE
Update README for scripts

### PR DESCRIPTION
**Pull request by ArturWieczorek**
Update README for scripts

Issue
-----------

- This PR **results**/**does not result** in breaking changes to upstream dependencies.

Checklist
---------
- [ ] This PR contains all the work required to resolve the linked issue.

- [ ] The work contained has sufficient documentation to describe what it does and how to do it.

- [ ] The work has sufficient tests and/or testing.

- [ ] I have committed clear and descriptive commits. Be considerate as somebody else will have to read these.

- [ ] I have added the appropriate labels to this PR.

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -54,21 +54,20 @@
 
 - <span><b>chairman.sh</b></span> <br/>  Run the Chairman consensus checker against a cluster started
                      by `shelley-testnet*.sh` family of scripts
-- <span><b>generator.sh</b></span> <br/> Run the transaction generator against a cluster started
```
Comment (ArturWieczorek) [inline scripts/README.md:57]: Generator was removed by @CodiePP  (there is a new `cardano-benchmarking` repo) so I also removed `Run transaction generator` - line 180

File: scripts/README.md [hunk 2]
```diff
@@ -54,21 +54,20 @@
...
-                     by `shelley-testnet` family of scripts
 - <span><b>genesis.sh</b></span> <br/> Generate a new genesis in the `configuration/` folder
-- <span><b>get-default-key-address.sh</b></span> <br/> Given a signing key, print its associated address
```
Comment (ArturWieczorek) [inline scripts/README.md:60]: This script was removed, but its functionality lives on as a shell function in `scripts/lib-cli.sh`

File: scripts/README.md [hunk 3]
```diff
@@ -54,21 +54,20 @@
...
 - <span><b>mainnet.sh</b></span> <br/> Run a node against Cardano Mainnet
+- <span><b>mainnet-via-fetcher.sh</b></span> <br/> Works in two phases, 1) prefetch - preload the prefetcher's 
+	 ChainDB, up to desired limit and 2) benchmark - run the second node as the benchmark, up to desired 
+	 slot limit 
 - <span><b>shelley-testnet-live.sh</b></span> <br/> Start a dev cluster with 3 nodes,
          with neat curses-based UI (run from tmux)
 - <span><b>shelley-testnet-dns.sh</b></span> <br/> Start a dev cluster with 3 nodes (run from tmux)
 - <span><b>shelley-testnet.sh</b></span> <br/> Start a dev cluster with 3 nodes, with
          basic logging (run from tmux)
-- <span><b>start-node.sh</b></span> <br/> Basic individual node startup script
```
Comment (ArturWieczorek) [inline scripts/README.md:71]: It doesn't exist, it was replaced by other scripts.

File: scripts/README.md [hunk 4]
```diff
@@ -109,15 +108,12 @@ set-window-option -g mouse on
 set -g default-terminal "tmux-256color"
 ```
 
-1.) create a `tmux` session
 
-`tmux new-session -s Demo`
```
Comment (ArturWieczorek) [inline scripts/README.md:114]: There is no longer a need to create a tmux session manually, see: https://github.com/input-output-hk/cardano-node/commit/d94c3194261d79dba931f1ac9872369032eabb0e

**Comment by ArturWieczorek** [comment]
The part of **README**: 
`See cardano-cli's [Transactions](../cardano-node/README.md#transactions) section for instructions on how to generate a tx.`
points to empty doc inside `cardano-node` repo, is that location still valid and that other README needs to be updated with some info or the link should point to `cardano-benchamrking` repo  @CodiePP ?

File: scripts/README.md [hunk 5]
```diff
@@ -54,21 +54,20 @@
...
                      by `shelley-testnet*.sh` family of scripts
-- <span><b>generator.sh</b></span> <br/> Run the transaction generator against a cluster started
-                     by `shelley-testnet` family of scripts
 - <span><b>genesis.sh</b></span> <br/> Generate a new genesis in the `configuration/` folder
-- <span><b>get-default-key-address.sh</b></span> <br/> Given a signing key, print its associated address
 - <span><b>issue-genesis-utxo-expenditure.sh</b></span> <br/> Write a file with a transaction
          spending a genesis UTxO entry, given a key owning it
 - <span><b>issue-utxo-expenditure.sh</b></span> <br/> Write a file with a transaction spending a
          normal UTxO entry, given a key owning it
 - <span><b>mainnet.sh</b></span> <br/> Run a node against Cardano Mainnet
```
Comment (CodiePP) [inline scripts/README.md:62]:
```suggestion
- <span><b>mainnet.sh</b></span> <br/> Run a node against Cardano Mainnet (the [configuration file](https://github.com/input-output-hk/cardano-node/blob/master/configuration/defaults/mainnet/configuration.yaml) contains documentation how to change options and turn on/off features like "LiveView")
```

**Pull request merged**